styles/variables-silicon.less

```
@@ -555,7 +555,13 @@
 @agate-togglebutton-smallest-min-width: 108px;
 @agate-togglebutton-indicator-border-radius: 50%;
 @agate-togglebutton-indicator-position: 50% auto auto;
+@agate-togglebutton-huge-indicator-position: @agate-togglebutton-indicator-position;
+@agate-togglebutton-small-indicator-position: @agate-togglebutton-indicator-position;
+@agate-togglebutton-smallest-indicator-position: @agate-togglebutton-indicator-position;
 @agate-togglebutton-indicator-height: 15px;
+@agate-togglebutton-huge-indicator-height: 15px;
+@agate-togglebutton-small-indicator-height: 15px;
+@agate-togglebutton-smallest-indicator-height: 15px;
```

I think we can use @agate-togglebutton-indicator-height variable. 
```suggestion
@agate-togglebutton-huge-indicator-height: @agate-togglebutton-indicator-height;
@agate-togglebutton-small-indicator-height: @agate-togglebutton-indicator-height;
@agate-togglebutton-smallest-indicator-height: @agate-togglebutton-indicator-height;
```